build/build.ps1

```diff
@@ -22,6 +22,7 @@ Param(
 
 Set-StrictMode -Version 2.0
 $ErrorActionPreference = "Stop"
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
The fix I saw in C# was to `|` Tls12 on to the defaults. Should we do that here?

Reply:
I copied this line verbatim from @jaredpar's fix.

Reply:
https://github.com/dotnet/roslyn/pull/25053/files

Reply:
I copied from @tmat 

https://github.com/dotnet/roslyn-tools/pull/169

Reply:
I guess we can OR it.